### ISSUE TITLE
Support for GitHub Enterprise?

### ISSUE THREAD

**Issue by copumpkin** [davido/gerrit-oauth-provider#48]
A quick glance suggests that the GitHub API endpoints are hardcoded to the public ones. Would it be possible to support custom API endpoints to use GitHub Enterprise, which otherwise behaves just like the public one?

**Comment by davido** [comment]
This feature is on my TODO list. I will try to implement it, unless someone else is faster than me.

**Comment by dgags** [comment]
Hey, any movement on this?

**Comment by davido** [comment]
@lucamilanesio: Any comments on that?

**Comment by lucamilanesio** [comment]
@davido yes, it should be fairly simple. It supported on the GitHub plugin already, just borrow the code from there ;-)

**Comment by jhahn21** [comment]
any progress on this? if not, I can help.

**Comment by davido** [comment]
The CL for GHE support was already uploaded for review: [1]. You could try that change and report back your feedback.

* [1] https://gerrit-review.googlesource.com/c/plugins/oauth/+/220372

**Comment by andred** [comment]
https://gerrit-review.googlesource.com/c/plugins/oauth/+/220372 works for me on an existing 2.15.13 installation. (Can't test against latest gerrit release, but I cherry-picked that change to the relevant branch)

**Comment by davido** [comment]
Thanks for confirming.
